packages/connector-logto-email/README.md

```diff
@@ -0,0 +1,5 @@
+# Logto email connector
```

Comment:
double check if we have code logic in core to remove logto connectors from normal installs (like mock connectors)

Reply:
Yes, see https://github.com/logto-io/logto/pull/3414